stresstests/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "azure-iot-stresstests",
+  "description": "Azure IoT Stress Tests",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0",
+  "private": true,
+  "license": "MIT",
+  "//": "TODO: will the bump version pipeline update the dependencies?",
```

Comment:
I was just going to add a comment. This directory needs to be added to internals/release/node-sdk/inputs.js